tourney/stats.py

```diff
@@ -335,10 +335,14 @@ def __fmt_num(self, num):
   def __fmt_top(self, lst, top_range, lookup):
     """Expects that `self.__personal` has already been filled!"""
     res = ""
+
+    req_plays = max([p["total_matches"] for p in self.__personal.values()]) / 4
+    qualifying_lst = [p for p in lst if self.__personal[p[0]]["total_matches"] > req_plays]
```

Comment:
Having the exact number of required plays should still be accepted:
```suggestion
    qualifying_lst = [p for p in lst if self.__personal[p[0]]["total_matches"] >= req_plays]
```

Reply:
Oh yeah, that seems fair